src/styles/nav-style.scss

```diff
@@ -0,0 +1,33 @@
+nav {
+    display: flex;
+    box-sizing: border-box;
+    flex-direction: column;
+    overflow: hidden;
+    background-color: #cbe9e3;
+    width: 100%;
+}
+
+.nav-logo {
+    width: 300px;
+}
+
+.nav-list {
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+}
+
+.sticky {
+    position: fixed;
+    top: 0;
+}
+
+.content {
+    padding: 0;
+}
+
+@media (min-width: 500px) {
```

Comment:
we could make a media file with const of break-points and import it in all css files